source/compiler/sc4.c

```diff
@@ -343,17 +343,19 @@ SC_FUNC void markexpr(optmark type,const char *name,cell offset)
  *
  *  Global references: funcstatus  (referred to only)
  */
-SC_FUNC void startfunc(char *fname)
+SC_FUNC void startfunc(char *fname,int generateproc)
 {
-  stgwrite("\tproc");
+  if(generateproc) {
```

Comment:
The coding style followed by the project requires a space between `if`, `while`, `switch` etc. and an opening parenhesis (e.g. `if (expr)`, `while (expr)`, `switch (expr)`).